pkgs/development/compilers/emscripten/cc.nix

```diff
@@ -0,0 +1,55 @@
+{ emscripten
+, python3, nodejs, gnumake, coreutils, pkg-config
+, stdenvNoCC, lib, writeScript, makeWrapper
+, runtimeShell
+, wasmArch ?
+  if stdenvNoCC.targetPlatform.isWasm
+  then stdenvNoCC.targetPlatform.uname.processor
+  else "wasm32"
+}:
+
+let
+  stdenv = stdenvNoCC;
+  targetPrefix = "${wasmArch}-emscripten-";
+  llvm = emscripten.llvmEnv;
+
+in stdenv.mkDerivation {
+  pname = "${wasmArch}-emscripten-cc";
+  version = emscripten.version;
+
+  preferLocalBuild = true;
+
+  passthru = {
+    isGNU = false;
+    isClang = true;
+    isEmscripten = true;
+    inherit targetPrefix nodejs emscripten;
+  };
+
+  dontBuild = true;
+  dontConfigure = true;
+  enableParallelBuilding = true;
+
+  buildInputs = [ makeWrapper ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

Reply:
Thanks! I missed your comment and wasted a few hours finding this bug.